pkgs/development/tools/build-managers/alibuild/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, python}:
+
+python.pkgs.buildPythonPackage rec {
+  pname = "alibuild";
+  version = "1.5.3";
+
+  src = python.pkgs.fetchPypi {
+    inherit pname version;
+    sha256 = "1am50qvpjh6cl3jf3bb66ikw73pmv64cpddcvgi99fy9zbq9iwlg";
+  };
+
+  argparse = null;
+
+  doCheck = false;
+  propagatedBuildInputs = [
+    python.pkgs.requests
+    python.pkgs.argparse
+    python.pkgs.pyyaml
+  ];
+
+  patches = [
+    ./fix-argparse.patch
```

Comment:
If this patch is not a nix-specific problem, please report to upstream and add a comment linking to the report.